modules/ROOT/pages/reference-mcp-tools.adoc

```diff
@@ -0,0 +1,79 @@
+= MuleSoft MCP Server Tools
+
+These tools are available in MuleSoft MCP Server. Some tools are available only in the MuleSoft MCP Server that's included in Anypoint Code Builder IDE; those tool descriptions include "Available only in Anypoint Code Builder."
+
+== Agent Network Development
+* `configure_agent_network_yaml`: Modifies an agent network project YAML file. The tool can replace placeholder YAML created from a template with values generated from user prompts and guided setup (mode='configure'). Or, the tool can make minor changes to a preconfigured agent network YAML file (mode='edit'). The LLM determines the appropriate mode value from user prompts and guided setup. Available only in Anypoint Code Builder.
+
+* `create_agent_network_project`: Create and scaffold an agent network project with a name and location you specify. Available only in Anypoint Code Builder.
+
+* `deploy_agent_network`: Deploy an agent network instance to a runtime environment. Specify the project path, target environment, private space, and Managed Flex Gateway configuration for deployment. Prerequisites: Managed Flex Gateway, a business group, and the agent network project contains a valid `exchange.json` file. Available only in Anypoint Code Builder.
```

Comment:
Private space should be replaced by `deployment target` since RTF is also supported